### PR TITLE
refactor: move apex test run to apex package

### PR DESCRIPTION
**Pull request by brpowell**
### What does this PR do?

Migrates the apex test run command to the apex package.

### What issues does this PR fix or reference?

@W-8660081@

### CODE REVIEW

File: packages/salesforcedx-vscode-apex/src/commands/forceApexTestRun.ts [hunk 1]
```diff
@@ -183,7 +188,7 @@ export class ApexLibraryTestRunExecutor extends LibraryCommandletExecutor<
 
   constructor() {
     super(
-      nls.localize('apex_test_run_text'),
+      nls.localize('force_apex_test_run_text'),
       'force_apex_execute_library',
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode-apex/src/commands/forceApexTestRun.ts:192]: let's change the log name parameter here to be something like `force_apex_test_run_library`

Comment (brpowell) [reply packages/salesforcedx-vscode-apex/src/commands/forceApexTestRun.ts:192]: lol whoops good catch

Comment (brpowell) [inline packages/salesforcedx-vscode-apex/src/commands/forceApexTestRun.ts:192]:
```suggestion
      'force_apex_test_run_library',
```

File: packages/salesforcedx-vscode-apex/src/commands/forceApexTestRun.ts [hunk 2]
```diff
@@ -162,8 +163,12 @@ function getTempFolder(): string {
 export class ForceApexTestRunExecutor extends SfdxCommandletExecutor<
   ApexTestQuickPickItem
 > {
+  constructor() {
+    super(OUTPUT_CHANNEL);
+  }
+
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode-apex/src/commands/forceApexTestRun.ts:169]: We should introduce the output channel changes for the `forceApexTestRunCodeAction.ts` file as well. If you run tests through the side bar or the code lens within the files, the output is displayed in the `Salesforce CLI` channel. Running the tests through the command palette displays the output in the `Apex Extension` channel - same as the other Apex commands. We should keep this consistent so users aren't confused on where to look for their test results

**Comment by brpowell** [comment]
Screen Shot 2021-02-05 at 1 56 08 PM

@ekapner Side note: is adding the word "Extension" at the end of these output channels superfluous? Other extensions don't seem to do that...wondering if it should just be "Apex" or "LWC"

File: packages/salesforcedx-vscode-apex/src/views/testRunner.ts [hunk 3]
```diff
@@ -74,11 +77,11 @@ export class ApexTestRunner {
...
-        channelService.appendLine('-----------------------------------------');
-        channelService.appendLine(stackTrace);
-        channelService.appendLine(errorMessage);
-        channelService.appendLine('-----------------------------------------');
-        channelService.showChannelOutput();
+        OUTPUT_CHANNEL.appendLine('-----------------------------------------');
+        OUTPUT_CHANNEL.appendLine(stackTrace);
+        OUTPUT_CHANNEL.appendLine(errorMessage);
+        OUTPUT_CHANNEL.appendLine('-----------------------------------------');
+        OUTPUT_CHANNEL.show(true);
```
Comment (brpowell) [inline packages/salesforcedx-vscode-apex/src/views/testRunner.ts:84]: Fixed the test sidebar outputting to Salesforce CLI when you click on a failure as well.

**Review by AnanyaJha — APPROVED** [review]

**Review by lcampos — APPROVED** [review]

**Pull request merged**